### PR TITLE
[issue #155] Create inventaire publishers

### PR DESCRIPTION
**Pull request by jum-s**
minimal attempt of creating and updating inventaire publishers
addressing #155

### CODE REVIEW

File: server/controllers/entities/lib/properties/properties_values_constraints.coffee [hunk 1]
```diff
@@ -65,6 +73,8 @@ module.exports =
   'wdt:P569': bases.uniqueSimpleDay
   # date of death
   'wdt:P570': bases.uniqueSimpleDay
+  # inception
+  'wdt:P571': bases.uniqueSimpleDay
```
Comment (maxlath) [inline server/controllers/entities/lib/properties/properties_values_constraints.coffee]: it looks like fae5f7c mistakenly removed `wdt:P576` or am I missing something?

File: server/controllers/entities/lib/properties/properties_config_bases.coffee [hunk 2]
```diff
@@ -25,6 +25,7 @@ module.exports =
   workEntity: restrictedEntityType 'work'
   serieEntity: restrictedEntityType 'serie'
   humanEntity: restrictedEntityType 'human'
+  countryEntity: restrictedEntityType 'countrie'
```
Comment (maxlath) [inline server/controllers/entities/lib/properties/properties_config_bases.coffee]: what about using `country` as the singular form? I think the added cost is acceptable, see proposed implementation d5abf5a

File: server/lib/wikidata/aliases.coffee [hunk 3]
```diff
@@ -82,14 +82,20 @@ module.exports = typesAliases =
     # bandes dessinées publishing company
     'wd:Q3279251'
   ]
-
+  # Types required to update the entity search engine
+  # see server/controllers/entities/lib/update_search_engine
```
Comment (maxlath) [inline server/lib/wikidata/aliases.coffee]: this comment should indeed be here now, but should removed above `publishers`

File: server/controllers/entities/lib/properties/properties_values_constraints.coffee (hunk 1 above)
Comment (jum-s) [reply server/controllers/entities/lib/properties/properties_values_constraints.coffee]: not really mistakenly, i decided it wasnt a relevant information enough to be displayed so i deleted without explaining my motivation...

File: server/controllers/entities/lib/properties/properties_config_bases.coffee (hunk 2 above)
Comment (jum-s) [reply server/controllers/entities/lib/properties/properties_config_bases.coffee]: it hurts my eyes less to see `countrie` here, than a `if pluralType is 'countries' then 'country'` 
its a hack but at least it stays with the other hardcoded types above

File: server/controllers/entities/lib/properties/properties_values_constraints.coffee (hunk 1 above)
Comment (jum-s) [reply server/controllers/entities/lib/properties/properties_values_constraints.coffee]: [for documentation purpose] could actually be useful for check publication editions from a post-mortem editor

File: server/controllers/entities/lib/properties/properties_values_constraints.coffee [hunk 4]
```diff
@@ -31,14 +31,20 @@ module.exports =
...
+  # owned by
+  'wdt:P127': bases.entity
   # movement
   'wdt:P135': bases.entity
   # genre
   'wdt:P136': bases.entity
   # based on
   'wdt:P144': bases.workEntity
+  # headquarters location
+  'wdt:P159': bases.entity
```
Comment (maxlath) [inline server/controllers/entities/lib/properties/properties_values_constraints.coffee]: this should probably also be let aside (as for `wdt:P17`) as we can't properly validate a location entity at the moment

File: server/controllers/entities/lib/properties/properties_values_constraints.coffee [hunk 5]
```diff
@@ -123,6 +133,8 @@ module.exports =
   'wdt:P2963': builders.externalId /^[1-9]\d*$/
   # GoodReads book ID
   'wdt:P2969': builders.externalId /^[1-9]\d*$/
+  # ISBN publisher prefix
+  'wdt:P3035': builders.externalId /^978-\d-\d{1,}$/
```
Comment (maxlath) [inline server/controllers/entities/lib/properties/properties_values_constraints.coffee]: allow `979` prefixes and be more restrictive on the following blocks (copying [P3035#P1793](https://www.wikidata.org/wiki/Property:P3035#P1793) regex)
```suggestion
  'wdt:P3035': builders.externalId /^97(8|9)-\d{1,5}-\d{2,7}$/
```

File: server/controllers/entities/search_type.coffee [hunk 6]
```diff
@@ -20,6 +20,7 @@ indexedTypes = [
   'movements'
   'publishers'
   'collections'
+  'countries'
```
Comment (maxlath) [inline server/controllers/entities/search_type.coffee]: `countries` can be removed for now

**Pull request merged**